.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "harfbuzz-sys/harfbuzz"]
 	path = harfbuzz-sys/harfbuzz
 	url = https://github.com/harfbuzz/harfbuzz
+[submodule "harfbuzz-sys/freetype-sys"]
+	path = harfbuzz-sys/freetype-sys
+	url = https://github.com/PistonDevelopers/freetype-sys
```

Comment:
I'm not sure this change belongs here. Can we simply add the configuration define for now?

Reply:
android builds fail without this

Reply:
harfbuzz-sys has a feature called bundled. But you cannot use the bundled feature with FT enabled unless you link the submodule in the tree since you need the headers. This is needed for platforms where FT is not installed like android.